tree-diff

```diff
@@ -1,10 +1,22 @@
 #!/usr/bin/env python3
 
 import argparse
+import hashlib
 import json
 import os
 
 
+def hash_file(fd):
+    BLOCK_SIZE = 65536
```

Comment:
Did you measure this? I used 4k elsewhere (one page). If it makes a difference, we should use the same everywhere :)

Reply:
I did no measures, I've changed it to 4096 to be consistent. We can do some benchmarks later if needed. :)